src/shared/containers/challenge-detail/index.jsx

```diff
@@ -970,8 +969,8 @@ const mapDispatchToProps = (dispatch) => {
             } else dispatch(a.dropCheckpoints());
           } else dispatch(a.dropCheckpoints());
           if (ch.status === CHALLENGE_STATUS.COMPLETED) {
-            dispatch(a.loadResultsInit(ch.legacyId));
-            dispatch(a.loadResultsDone(tokens, ch.legacyId, ch.track.toLowerCase()));
+            dispatch(a.loadResultsInit(ch.id));
+            dispatch(a.loadResultsDone(ch.id));
```

Comment:
@jmgasper why are the other arguments `loadResultsDone` missing?
Is `loadResultsDone` moved to the react-lib and updated thus not to see it here?

Reply:
The tokens weren't used in the original call, and the track isn't necessary since we can now get that from the v5 challenge object response.